Cake.Android.SdkManager/BuildToolsParser.cs

```diff
@@ -0,0 +1,107 @@
+﻿using Cake.Core.IO;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text.RegularExpressions;
+
+namespace Cake.AndroidSdkManager
+{
+	internal static class BuildToolsParser
+	{
+		internal static AndroidSdkManagerList ParseSdkManagerList(IProcess process)
+		{
+			var result = new AndroidSdkManagerList();
+
+			int section = 0;
+			bool isDependencies = false;
+			var bufferedLines = new Stack<string>();
+
+			foreach (var line in process.GetStandardOutput())
+			{
+				if (line.ToLowerInvariant().Contains("installed packages:"))
```

Comment:
All these calls to `ToLowerInvariant` could be already set to a new variable like `lineInLower` or you could use a for loop and set the variable line to `array[i].ToLowerInvariant()` :)
It would make the code more readable